src/main/java/Pool.java

```diff
@@ -0,0 +1,19 @@
+import vi.viredis.client.ViRedis;
+
+public interface Pool<T> {
```

Comment:
This is very inconsistent. You define the interface Pool for type T, but when you define get() you only return one specific type: ViRedis.

Also: there is no documentation for the interface. What _is_ this Pool? What sets it apart from all the other queues, what makes it so special that it deserves a class definition of its own. What is expected from the classes that will implement this interface?

Reply:
done!